packages/hoppscotch-common/src/components/app/SpotlightSearch.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+  <div
+    class="border-animation relative p-[1px] rounded flex-1 self-stretch overflow-hidden flex items-center justify-center"
+    :class="{
+      'before:top-1/2 before:left-1/2 before:-translate-x-1/2 before:-translate-y-1/2 before:aspect-square before:w-full before:absolute before:bg-':
+        !HAS_OPENED_SPOTLIGHT,
+    }"
+    aria-hidden="true"
+  >
+    <button
+      class="relative flex flex-1 cursor-text items-center justify-between self-stretch rounded bg-primaryDark px-2 text-secondaryLight transition hover:border-dividerDark hover:bg-primaryLight hover:text-secondary focus-visible:border-dividerDark focus-visible:bg-primaryLight focus-visible:text-secondary overflow-hidden"
+      @click="
+        () => {
+          invokeAction('modals.search.toggle', undefined, 'mouseclick')
+          !HAS_OPENED_SPOTLIGHT ? toggleSetting('HAS_OPENED_SPOTLIGHT') : null
+        }
+      "
+    >
+      <span class="inline-flex flex-1 items-center">
+        <icon-lucide-search class="svg-icons mr-2" />
+        <span v-if="!HAS_OPENED_SPOTLIGHT" class="flex flex-1">
+          Try
+          <TransitionGroup tag="div" name="list" class="ml-1 relative">
+            <span
+              v-for="(word, index) in wordToShow"
```

Comment:
We are showing a phrase/sentence here.  A better naming would be `phrasesToShow` in my opinion.